lib/sass/calcite-web/patterns/_accordion.scss

```diff
@@ -5,11 +5,21 @@
 //  ↳ patterns → _accordion.md
 
 @mixin accordion() {
-  @include side-nav();
 }
+
   @mixin accordion-section() {
+    border: 1px solid $light-gray;
+    margin: -1px 0 0 0;
+    color: $dark-gray;
+    @include transition(all .15s ease-in-out);
+    &:focus,
+    &:hover {
+      border-color: $dark-gray;
+      color: $off-black;
+      position: relative;
```

Comment:
I think this could be with the normal styles, and doesn't have to be declared for focus and hover